backend/api/manageUsersApi.test.js

```diff
@@ -58,4 +58,19 @@ describe('manageUsersApi tests', () => {
       expect(client.put).toBeCalledWith(context, '/roles/role1/description', roleDescription)
     })
   })
+
+  describe('change role admin type', () => {
+    const roleAdminType = { adminType: ['DPS_ADM'] }
+
+    beforeEach(() => {
+      client.put = jest.fn().mockReturnValue({
```

Comment:
You can also use `mockResolvedValue` which will handle the promise type behaviour.